mne/tests/test_source_space.py

```diff
@@ -55,9 +60,48 @@
 
 base_dir = op.join(op.dirname(__file__), '..', 'io', 'tests', 'data')
 fname_small = op.join(base_dir, 'small-src.fif.gz')
+fname_ave = op.join(base_dir, 'test-ave.fif')
 rng = np.random.RandomState(0)
 
 
+@testing.requires_testing_data
+@pytest.mark.parametrize('picks, limits', [
+    ('meg', (0.020, 0.250)),
+    (None, (0.002, 0.250)),  # should be same as EEG
+    ('eeg', (0.002, 0.250)),
+])
+def test_compute_distance_to_sensors(picks, limits):
+    """Test computation of distances between vertices and sensors."""
+    src = read_source_spaces(fname_fs)
+    info = read_info(fname_ave)
+    trans = read_trans(trans_fname)
+    if isinstance(picks, str):
+        kwargs = dict()
+        kwargs[picks] = True
+        if picks == 'eeg':
+            info['dev_head_t'] = None  # should not break anything
+        use_picks = pick_types(info, **kwargs, exclude=())
+    else:
+        use_picks = picks
+    n_picks = len(_picks_to_idx(info, use_picks, 'data', exclude=()))
+    n_verts = src[0]['nuse'] + src[1]['nuse']
+
+    # minimum distances between vertices and sensors
+    depths = compute_distance_to_sensors(src, info=info, picks=use_picks,
+                                         trans=trans)
+    assert depths.shape == (n_verts, n_picks)
+    assert limits[0] * 2 > depths.min()  # meaningful choice of limits
+    assert_array_less(limits[0], depths)
+    assert_array_less(depths, limits[1])
+
+    if picks != 'eeg':
+        # this should break things
+        info['dev_head_t'] = None
+        with pytest.raises(ValueError,
+                           match='Transform between meg<->head'):
+            compute_distance_to_sensors(src, info, use_picks, trans)
```

Comment:
you don't seem to test when not passing a trans file which is optional in your API